pycbc/strain/strain.py

```diff
@@ -1443,6 +1474,11 @@ def __init__(self, frame_src, channel_name, start_time,
         self.add_hard_count()
         self.taper_immediate_strain = True
 
+        # Caches for FFTs to use class based API
+        self.make_freq_cache = {}
```

Comment:
This naming is pretty opaque. Any way to name this in a way that makes it clear which cache is doing what?

Reply:
This was changed in response to below comment.